flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/job/JobDetailsInfo.java

```diff
@@ -59,6 +59,8 @@
 
     public static final String FIELD_NAME_DURATION = "duration";
 
+    public static final String FIELD_NAME_MAX_PARALLELISM = "job-max-parallelism";
```

Comment:
same as in other files; remove "job" prefix and adjust case

Reply:
Done